js/scripts/honorSystem/honorSystem.js

```diff
@@ -0,0 +1,102 @@
+/**
+ * Initializes the honor at the start of a game to default honor 
+ * unless honor values already exists in currentHonor.
+ * This class is intended to modify some of CHAD's stats based on his honor and the honor points gained
+ * or lost during a particular event or action. 
+ * The constructor takes in a positive or negative value for delatahonorPts from at least 0 to 5 points.
+ * @author Caleb Krauter
+ */
+
+class honorSystem {
+    constructor(deltaHonorPts, isNewGame) {
```

Comment:
Constructor needs proper JSDoc commenting. It is unclear what type deltaHonorPts and isNewGame should be, nor do I understand what they should be.